### PR TITLE
EDU-1087: add note about tctl deprecation and link CLI release notes

### PR DESCRIPTION
**Pull request by DJSanti**
## What does this PR do?
This PR adds a note to the CLI index about the deprecation of `tctl`, and refers users to the CLI release notes for correcting env and command names. Corrections were made to convert passive voice to active voice.

## Notes to reviewers
@aarohib You'd be the best for verifying the placement and accuracy of this note.

### CODE REVIEW

File: docs-src/concepts/what-is-the-temporal-cli.md [hunk 1]
```diff
@@ -41,30 +54,29 @@ The Temporal CLI is a command-line tool that includes a distribution of a Tempor
...
    - [Windows arm64](https://temporal.download/cli/archive/latest?platform=windows&arch=arm64)
 2. Extract the downloaded archive.
-3. Add the `temporal` binary to your PATH. (`temporal.exe` for Windows)
+3. Add the `temporal` binary to your PATH (use `temporal.exe` for Windows).
 
 ## Starting the Temporal Server
 
-Run the following command to start the Temporal Server.
-This command also starts the Web UI.
+To start the Temporal Server and Web UI, run the following command:
```
Comment (aarohib) [inline docs-src/concepts/what-is-the-temporal-cli.md:61]:
```suggestion
To start the Temporal Server, run the following command:
```

File: docs-src/concepts/what-is-the-temporal-cli.md [hunk 2]
```diff
@@ -242,80 +254,72 @@ OPTIONS:
...
-| `TEMPORAL_CLI_NAMESPACE`                     | Identifies a Namespace in the Temporal Workflow. Default value: "default".        | --namespace                     |
-| `TEMPORAL_CLI_TLS_CA`                        | Path to server CA certificate.                                                    | --tls-ca-path                   |
-| `TEMPORAL_CLI_TLS_CERT`                      | Path to x509 certificate.                                                         | --tls-cert-path                 |
-| `TEMPORAL_CLI_TLS_DISABLE_HOST_VERIFICATION` | Disables TLS host name verification if already enabled. Default value: false.     | --tls-disable-host-verification |
-| `TEMPORAL_CLI_TLS_KEY`                       | Path to private certificate key.                                                  | --tls-key-path                  |
-| `TEMPORAL_CLI_TLS_SERVER_NAME`               | Provides an override for the target TLS server name.                              | --tls-server-name               |
+Configure system environment with Temporal CLI environmental variables.
+Use the table below as reference:
```
Comment (aarohib) [inline docs-src/concepts/what-is-the-temporal-cli.md:305]:
```suggestion
The following table lists and describes the environment variables you can set for Temporal CLI.
```

**Comment by aarohib** [comment]
Thank you for taking this on so quickly! Left a couple of comments, but overall looks great.

**Comment by DJSanti** [comment]
@aarohib Made suggested changes. Sorry for the delay!

**Review by aarohib — APPROVED** [review]

File: docs-src/concepts/what-is-the-temporal-cli.md [hunk 3]
```diff
@@ -8,7 +8,14 @@ tags:
   - cli
 ---
 
-The Temporal CLI is a command-line tool that includes a distribution of a Temporal Cluster ([Temporal Server](/concepts/what-is-the-temporal-server), persistence (SQLite), and the [Temporal Web UI](/concepts/what-is-the-temporal-web-ui)).
+:::note
+
+When upgrading from `tctl` to CLI, make sure to update your environment variables and use updated commands.
```
Comment (djmagee) [inline docs-src/concepts/what-is-the-temporal-cli.md:13]:
```suggestion
When upgrading from `tctl` to Temporal CLI, make sure to update your environment variables and use updated commands.
```

File: docs-src/concepts/what-is-the-temporal-cli.md [hunk 4]
```diff
@@ -41,30 +54,29 @@ The Temporal CLI is a command-line tool that includes a distribution of a Tempor
...
-Run the following command to start the Temporal Server.
-This command also starts the Web UI.
+To start the Temporal Server, run the following command:
 
 ```bash
 temporal server start-dev
 ```
 
-At this point you should have a server running on `localhost:7233` and a web interface at <http://localhost:8233>.
+This launches a server on `localhost:7233` and a web interface at <http://localhost:8233>.
```
Comment (djmagee) [inline docs-src/concepts/what-is-the-temporal-cli.md:67]:
```suggestion
The command launches a server on `localhost:7233` and a web interface at <http://localhost:8233>.
```

File: docs-src/concepts/what-is-the-temporal-cli.md [hunk 5]
```diff
@@ -41,30 +54,29 @@ The Temporal CLI is a command-line tool that includes a distribution of a Tempor
...
 
-By default, it doesn’t persist your data—if you start a Workflow, Ctrl-C, and run the command again, your Workflow will be gone.
-
-If you'd like your Workflows to be saved, use the `--db-filename` flag:
+By default, data isn't persisted.
+If you want to save Workflows, use the `--db-filename` flag:
```
Comment (djmagee) [inline docs-src/concepts/what-is-the-temporal-cli.md:70]:
```suggestion
If you want to save Workflows, use the `--db-filename` option:
```

File: docs-src/concepts/what-is-the-temporal-cli.md [hunk 6]
```diff
@@ -160,7 +172,7 @@ $ temporal workflow list --fields long --output json
 ]
 ```
 
-Filter out just the type with [jq](https://stedolan.github.io/jq/):
+Filter out Workflows based on Type with [jq](https://stedolan.github.io/jq/):
```
Comment (djmagee) [inline docs-src/concepts/what-is-the-temporal-cli.md:175]:
```suggestion
Filter out Workflows based on Workflow Type with [jq](https://stedolan.github.io/jq/):
```
Or can simply lowercase "type" in this instance. Either way is good.

File: docs-src/concepts/what-is-the-temporal-cli.md [hunk 7]
```diff
@@ -179,18 +191,18 @@ $ temporal workflow list --fields long -o json | jq '.[].type.name' | uniq -c
    2 "MyWorkflow"
 ```
 
-To see what else you can do to Workflows, run `temporal workflow` or visit [CLI ▶️ workflow](/cli/workflow).
+To see the full range of Workflow-related commands, run `temporal workflow` or visit [CLI ▶️ workflow](/cli/workflow).
```
Comment (djmagee) [inline docs-src/concepts/what-is-the-temporal-cli.md:194]:
```suggestion
To see the full range of Workflow-related commands, run `temporal workflow` or see the [Temporal CLI workflow command reference](/cli/workflow).
```
Per https://developers.google.com/style/cross-references#references-to-other-documents.

File: docs-src/concepts/what-is-the-temporal-cli.md [hunk 8]
```diff
@@ -179,18 +191,18 @@ $ temporal workflow list --fields long -o json | jq '.[].type.name' | uniq -c
...
 
-To see the list of top-level commands, run `temporal` or visit [Available commands](#available-commands).
+For a full list of available commands, run `temporal` or visit [Available commands](#available-commands).
```
Comment (djmagee) [inline docs-src/concepts/what-is-the-temporal-cli.md:196]:
```suggestion
For a full list of available commands, run `temporal` or see [Available commands](#available-commands).
```

File: docs-src/concepts/what-is-the-temporal-cli.md [hunk 9]
```diff
@@ -327,27 +331,21 @@ Add the following code snippet to your `~/.zshrc` file:
...
 
-If you're running auto-completion from the terminal, run the following command:
+If you're running auto-completion from the terminal, run:
 
 ```sh
 echo 'source <(temporal completion zsh)' >> ~/.zshrc
 ```
 
-After setting the variable, run the following command:
+After setting the variable, run:
```
Comment (djmagee) [inline docs-src/concepts/what-is-the-temporal-cli.md:340]:
```suggestion
After setting the variable, run the following command:
```

File: docs-src/concepts/what-is-the-temporal-cli.md [hunk 10]
```diff
@@ -327,27 +331,21 @@ Add the following code snippet to your `~/.zshrc` file:
 source <(temporal completion zsh)
 ```
 
-If you're running auto-completion from the terminal, run the following command:
+If you're running auto-completion from the terminal, run:
```
Comment (djmagee) [inline docs-src/concepts/what-is-the-temporal-cli.md:334]:
```suggestion
If you're running auto-completion from the terminal, run the following command:
```
Per https://developers.google.com/style/colons#introductory-phrase-preceding-colon.

File: docs-src/concepts/what-is-the-temporal-cli.md [hunk 11]
```diff
@@ -327,27 +331,21 @@ Add the following code snippet to your `~/.zshrc` file:
...
 ### Bash auto-completion
 
-Bash auto-completion relies on `bash-completion`.
-
-Install the software with the steps provided in the [bash-completion README](https://github.com/scop/bash-completion#installation), or use your preferred package manager on your operating system.
-
-For more information, see [the Bash Completion page on Repology](https://repology.org/project/bash-completion/versions).
-
-### Aliases
+Install []`bash-completion`](https://github.com/scop/bash-completion#installation) and add the software to your `~/.bash_profile`.
```
Comment (djmagee) [inline docs-src/concepts/what-is-the-temporal-cli.md:346]:
```suggestion
Install [bash-completion](https://github.com/scop/bash-completion#installation) and add the software to `~/.bash_profile`.
```

File: docs-src/concepts/what-is-the-temporal-cli.md [hunk 12]
```diff
@@ -327,27 +331,21 @@ Add the following code snippet to your `~/.zshrc` file:
...
 
-You can also add aliases to your `~/.bash_profile` such as:
+To utilize aliases, add them to `~/.bash_profile` as needed.
```
Comment (djmagee) [inline docs-src/concepts/what-is-the-temporal-cli.md:348]:
```suggestion
To use aliases, add them to `~/.bash_profile` as needed.
```

**Pull request merged**